politeiawww/records/process.go

```diff
@@ -230,7 +230,18 @@ func (r *Records) processSetStatus(ctx context.Context, ss v1.SetStatus, u user.
 		return nil, err
 	}
 	rc := convertRecordToV1(*pdr)
-	recordPopulateUserData(&rc, u)
+
+	// Get record to populate username.
+	record, err := r.record(ctx, ss.Token, ss.Version)
```

Comment:
You're doing a bunch of extra work here.  You already have the record.  I believe the fix for this is using the `r.convertRecordToV1()` method instead of the `convertRecordToV1`.  Looking back on this, my documentation of `r.convertRecordToV1()` sucks.